contents.json

```diff
@@ -20528,6 +20528,30 @@
       "stars": 1198,
       "date_added": "May 20 2020",
       "suggested_by": "@jasonblais"
+    },
+     {
+      "title": "Chatwoot",
+      "category-ids": [
+        "customer-support",
```

Comment:
Please remove thus category, it's not supported

Reply:
@dkhamsing Donw